decidim-core/lib/decidim/components/base_controller.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 module Decidim
-  module Components
+  module Features
```

Comment:
Wrong path name? This should be under `decidim-core/lib/decidim/features/`, right?